### ISSUE TITLE
Describe evergreen Update using tools like renovate

### ISSUE THREAD

**Issue by d-koppenhagen** [angular/angular#46118]
### Description

Often companies want to be able to have an evergreen environment with up-to-date dependencies to be able to fix issues and vulnerability ideally automatically by running update autonomous or at least providing automatic pull requests for such updates.

One very common tool for such update is [renovate](https://www.whitesourcesoftware.com/free-developer-tools/renovate/).

To help users setting up such tools, the angular docs should describe the steps needed/the commands to be executed to target such goal.

have a look at the related renovate issue/PR:

https://github.com/renovatebot/renovate/issues/4256
https://github.com/renovatebot/renovate/pull/7632

General steps:

- update and bundle all angular-related dependencies in the package.json/ lockfile
- install the dependencies
- execute migrations afterwards

 Example using renovate:

```json
{
  "postUpgradeTasks": {
    "commands": [
      "npm ci --ignore-scripts",
      "npx ng update {{{depName}}} --from={{{fromVersion}}} --to={{{toVersion}}} --migrateOnly --allowDirty --force"
    ],
    "fileFilters": ["**/**"]
  }
}
```

### What is the affected URL?

https://angular.io/cli/update

### Please provide the expected behavior vs the actual behavior you encountered

Evergreen updates execution process using automation tools is described

**Comment by alan-agius4** [comment]
It's important to point that `postUpgradeTasks` only works when using the self-hosted version of Renovate.

**Comment by d-koppenhagen** [comment]
Yes I agree! It would be necessary to group all this dependencies into one group. And probably.

**Comment by JeanMeche** [comment]
I'm going ahead and close this as this isn't something that is specific to angular, a basically any project  has this issue.
